deny.toml

```
@@ -98,8 +98,7 @@ allow-git = [
     "https://github.com/str4d/redjubjub",
 
     # ticket #2523: replace unmaintained multiset
-    "https://github.com/jmitchell/multiset",
-]
+    "https://github.com/jmitchell/multiset",]
```

Nit: unnecessary whitespace change

```suggestion
    "https://github.com/jmitchell/multiset",
]
```